UPGRADE-3.x.md

```diff
@@ -0,0 +1,8 @@
+UPGRADE 3.x
+===========
+
+### Tests
+
+All files under the ``Tests`` directory are now correctly handled as internal test classes. 
```

Comment:
This have to be fixed after https://github.com/sonata-project/dev-kit/issues/179#issuecomment-229606357.

cc @core23